packages/audit/package.json

```
@@ -13,8 +13,7 @@
   },
   "scripts": {
     "lint": "eslint -c ../../eslint.json src/**/*.ts test/**/*.ts",
-    "_test": "cd ../.. && c8 --reporter lcov --reports-dir packages/audit/coverage ts-node packages/audit/test --type-check",
-    "test": "pnpm run compile && pnpm run _test",
+    "test": "pnpx jest",
```

you can just use jest here. `pnpm run` adds the root `node_modules/.bin` to the PATH
```suggestion
    "test": "jest",
```